### PR TITLE
Restyle banner 379

### PR DESCRIPTION
**Pull request by thegentlemanphysicist**
- Adds the illustration to the digital framework page banner
- Retools the homepage banner so that banner with side img works as a common component
- Refactors css of digital framework page and deletes the digital.css file
- Implements new banner on resourses , products and services, and digital principle pages as well.

### CODE REVIEW

File: react-frontend/src/components/StyleComponents/headings.js [hunk 1]
```diff
@@ -8,6 +8,25 @@ export const CaseStudyHeading = styled.h2.attrs({
...
+  font-size: 22pt;
+  font-weight: bold;
+  line-height: 24pt;
+  margin-bottom: 5px;
+`;
+
+export const DigitalFrameworkSubHeading = styled.h3.attrs({
+  className: 'digitalFrameworkSubHeading',
+})`
+  font-size: 16pt;
```
Comment (mark-a-wilson) [inline react-frontend/src/components/StyleComponents/headings.js:23]: Please change **16pt** to **18px**

File: react-frontend/src/components/StyleComponents/htmlTags.js [hunk 2]
```diff
@@ -23,6 +24,26 @@ export const BreadcrumbUL = styled.ul`
   margin: 0;
 `;
 
+export const DigitalFrameworkP = styled.p.attrs({
+  className: 'digitalParagraph',
+})`
+  font-family: Arial, Helvetica, sans-serif;
```
Comment (mark-a-wilson) [inline react-frontend/src/components/StyleComponents/htmlTags.js:30]: Please change **font-family** to `‘BC Sans’, ‘Noto Sans’, Verdana, Arial, sans-serif;`

**Review by mark-a-wilson — DISMISSED** [review]
Just noticed two minor font issues which I've commented. Other than that, I'm good with this.

**Review by mark-a-wilson — APPROVED** [review]
Looks great. 👍 I'm loving the illustrations, @singhinderdeep!

File: react-frontend/src/components/DigitalPrinciples/digitalPrinciples.js [hunk 3]
```diff
@@ -1,26 +1,24 @@
...
+      <PageContainer>
         <GlobaStyleSize />
-        <SimpleTextPageTitle>
-          Digital Principles for the Government of British Columbia (Version
-          1.0)
-        </SimpleTextPageTitle>
-
+        <BannerSideImage
+          title={
+            'Digital Principles - Digital Government - Province of British Columbia'
```
Comment (mark-a-wilson) [inline react-frontend/src/components/DigitalPrinciples/digitalPrinciples.js:18]: Just noticed this - could you please change the title in the banner back to what it was: "Digital Principles for the Government of British Columbia (Version 1.0)?"

I know this is confusing, but this is one case where I think a more succinct title for the `<title>` works for its Accessibility function ... but the text in the `<h1>` needs to remain.

Thanks!

**Review by mark-a-wilson — APPROVED** [review]
Thanks for making that correction.

File: react-frontend/src/components/Resources/resources.js [hunk 4]
```diff
@@ -4,19 +4,23 @@ import ForDesigners from './fordesigners';
 import ForDevelopers from './fordevelopers';
 import DocumentTitle from 'react-document-title';
 import Guides from './guides';
-import SimpleBanner from '../PageElements/BannerSimple/bannerSimple';
+import BannerSideImage from '../PageElements/BannerSideImage/bannerSideImage';
```
Comment (parulmishra) [inline react-frontend/src/components/Resources/resources.js:7]: The BannerSideImage folder seems to contain only one file. Do we expect more files in this folder in future?

File: react-frontend/src/components/StyleComponents/headings.js [hunk 5]
```diff
@@ -8,6 +8,25 @@ export const CaseStudyHeading = styled.h2.attrs({
...
+
+export const DigitalFrameworkSubHeading = styled.h3.attrs({
+  className: 'digitalFrameworkSubHeading',
+})`
+  font-size: 18pt;
+  font-weight: bold;
+  line-height: 24pt;
+  margin-bottom: 5px;
+  margin-top: 5px;
+`;
```
Comment (parulmishra) [inline react-frontend/src/components/StyleComponents/headings.js:28]: Can we use style extensions here? 
https://styled-components.com/docs/basics#extending-styles

File: react-frontend/src/components/Resources/resources.js (hunk 4 above)
Comment (thegentlemanphysicist) [reply react-frontend/src/components/Resources/resources.js:7]: Good idea, the different banners should probably all be in one folder.

File: react-frontend/src/components/StyleComponents/headings.js (hunk 5 above)
Comment (thegentlemanphysicist) [reply react-frontend/src/components/StyleComponents/headings.js:30]: good idea, I was so focused on pulling things into styled components I didn't try simplifying them once they'd been moved. 
Implented a few, but I will probably have to do more once the rest of the css file have been removed. (I'll probably need to do the same with page content to make alignment a bit more robust.)

**Review by mark-a-wilson — APPROVED** [review]
If it helps gets this merged to develop, I just had another look at the deployment and it all looks good.

**Review by parulmishra — APPROVED** [review]

**Pull request merged**